src/MessageWindow.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import MessageGroup from './MessageGroup';
+import MessageText from './MessageText';
+import { MessageContainerType } from './MessengerTypes';
+import TypingIndicator from './TypingIndicator';
+
+class MessageWindow extends React.Component {
+    render() {
+        let groups = this.props.messagesGroup.map((group, index) => {
+            const isMyMessage = group.group === this.props.user;
+            return (
+                <div className='fxcol' key={index + 1}>
+                    <span className='user-name'>
+                        {!isMyMessage ? group.group : ''}{' '}
```

Comment:
Zamiast `? :` można użyć `{!isMyMessage && <span className='user-name'>{group.group}</span>`

Reply:
W sumie to takie proste, ale nie używałam nigdy w ten sposób takiej konstrukcji warunkowej. Podoba mi się :)